client/pov-recovery/src/lib.rs

```diff
@@ -380,6 +353,70 @@ where
 			.import_blocks(BlockOrigin::ConsensusBroadcast, incoming_blocks);
 	}
 
+	/// Attempts an explicit recovery of one or more blocks.
+	pub fn recover(&mut self, req: RecoveryRequest<Block>) {
+		let RecoveryRequest { mut hash, delay, kind } = req;
+		let mut to_recover = Vec::new();
+
+		let do_recover = loop {
+			let candidate = match self.candidates.get_mut(&hash) {
+				Some(candidate) => candidate,
+				None => {
+					tracing::error!(
```

Comment:
```suggestion
					tracing::debug!(
```

I mean when you just have restarted your node, this is expected behavior. The operator also can not really do anything against this.